diff --git a/Trees/Binary_Heap_Tree_UnitTests.py b/Trees/Binary_Heap_Tree_UnitTests.py
index 10a48e1..ebcfc70 100644
--- a/Trees/Binary_Heap_Tree_UnitTests.py
+++ b/Trees/Binary_Heap_Tree_UnitTests.py
@@ -5,7 +5,7 @@
 from Binary_Heap_Tree_Structure import BinaryHeap
 import unittest
 
-class TestBST(unittest.TestCase):
+class TestBinaryHeapTree(unittest.TestCase):
     
     def setUp(self):
         self.empty = BinaryHeap()
diff --git a/Trees/Trie_Prefix_Count_Speed_UnitTests.py b/Trees/Trie_Prefix_Count_Speed_UnitTests.py
index 20eead0..81e962c 100644
--- a/Trees/Trie_Prefix_Count_Speed_UnitTests.py
+++ b/Trees/Trie_Prefix_Count_Speed_UnitTests.py
@@ -5,7 +5,7 @@
 from Trie_Prefix_Count_Speed import Trie
 import unittest
 
-class TestBST(unittest.TestCase):
+class TestPrefixTrie(unittest.TestCase):
     
     def setUp(self):
         self.empty = Trie()
diff --git a/Trees/Trie_Prefix_Count_UnitTests.py b/Trees/Trie_Prefix_Count_UnitTests.py
index f920cc9..7319f2f 100644
--- a/Trees/Trie_Prefix_Count_UnitTests.py
+++ b/Trees/Trie_Prefix_Count_UnitTests.py
@@ -5,7 +5,7 @@
 from Trie_Prefix_Count import Trie
 import unittest
 
-class TestBST(unittest.TestCase):
+class TestPrefixTrie(unittest.TestCase):
     
     def setUp(self):
         self.empty = Trie()
diff --git a/Trees/Trie_Time_Tester.py b/Trees/Trie_Time_Tester.py
index 7da05f8..1932188 100644
--- a/Trees/Trie_Time_Tester.py
+++ b/Trees/Trie_Time_Tester.py
@@ -1,3 +1,7 @@
+#Michael Robertson
+#mirob2005@gmail.com
+#Completed: 3/5/2013
+
 import time
 from Trie_Prefix_Count import Trie as Trie_space
 from Trie_Prefix_Count_Speed import Trie as Trie_speed
diff --git a/Trees/Trie_UnitTests.py b/Trees/Trie_UnitTests.py
index b739c6b..1c94b04 100644
--- a/Trees/Trie_UnitTests.py
+++ b/Trees/Trie_UnitTests.py
@@ -5,7 +5,7 @@
 from Trie import Trie
 import unittest
 
-class TestBST(unittest.TestCase):
+class TestTrie(unittest.TestCase):
     
     def setUp(self):
         self.empty = Trie()
